barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/ultra_circuit_builder.hpp

```diff
@@ -872,10 +852,9 @@ template <typename FF> class UltraCircuitBuilder_ : public CircuitBuilderBase<ar
         size_t nnfcount = 0;
         get_num_gates_split_into_components(count, rangecount, romcount, ramcount, nnfcount);
 
-        size_t total = count + romcount + ramcount + rangecount + num_ecc_op_gates;
+        size_t total = count + romcount + ramcount + rangecount;
```

Comment:
Are we intending to remove num_ecc_op_gates?

Reply:
I'm about to delete this PR, but yes, this kind of change is intentional and will be present in the real PR. Part of this work is removing all goblin-related functionality from the UltraCircuitBuilder and into a new GoblinUltraCircuitBuilder. `num_ecc_op_gates` is a goblin-only object.